.npmignore

```diff
@@ -0,0 +1,7 @@
+!dist
```

Comment:
That's not necessary, everything which is not part of the rest of the list is not ignored anyway